egs2/TEMPLATE/svs1/README.md

```diff
@@ -0,0 +1,550 @@
+# ESPnet2 SVS Recipe TEMPLATE
+
+This is a template of SVS recipe for ESPnet2.
+
+## Table of Contents
+
+* [ESPnet2 SVS Recipe TEMPLATE](#espnet2-svs-recipe-template)
+  * [Table of Contents](#table-of-contents)
+  * [Recipe flow](#recipe-flow)
+    * [1\. Database-dependent data preparation](#1-database\-dependent-data-preparation)
+    * [2\. Standard audio and midi formatting](#2-standard-audio-and-midi-formatting)
```

Comment:
The subtitles in this part should be consistent with the following content so as to build the links.